### ISSUE TITLE
Training optimize.py produces tv_loss is `nan`

### ISSUE THREAD

**Issue by pjama** [lengstrom/fast-style-transfer#91]
While training a model using `udnie.jpg` style image, the test images appear blank, and the losses seem to be incorrect as well (without change, style loss: ~1e+24 and tv_loss: **nan**)

The command I am executing:

```shell
python style.py --style examples/style/udnie.jpg --checkpoint-dir data/checkpoints/udnie/ --test examples/content/chicago.jpg --test-dir test --checkpoint-iterations 1000 --batch-size 10
```

The specs of the training system:

* Ubuntu 16.04
* GeForce GTX 1080
* TensorFlow r1.0 (compiled)

Any thoughts about how to resolve the problem? Thanks in advance.

**Comment by ScorpioCPH** [comment]
@pjama I have the same result as you, Do you resolve this problem?

**Comment by pjama** [comment]
@ScorpioCPH – I found issue #40, which recommends using commit 55809f4. 

This appears to solve my issue, and the model trains properly now, however I was unable to find the cause of the behavior described above.

My debugging efforts led me to investigate the following lines of `optimize.py`:

```python
preds = transform.net(X_content/255.0)
preds_pre = vgg.preprocess(preds)
```

In my case, the `nan` values originated from the `preds` variable.

**Comment by ScorpioCPH** [comment]
@pjama Thanks! After trying `55809f4`, It is ok for me too.

I think you can file a PR to fix it if possible :-)
